01_fib/fibonacci_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Add `TestCalculateFib` and compare expected and actual slices.
Test corner cases - n: -1, 0 ,-1, very large number.